content/en/docs/examples/microservices-istio/prereq/index.md

```diff
@@ -18,7 +18,8 @@ computer.
 
 Ensure the following conditions are met:
 
-- You have administrator privileges on a Kubernetes cluster.
+- You have administrator privileges on a Kubernetes cluster named
```

Comment:
you also need admin on the vm/bm on which the K8s cluster is running - to modify `/etc/hosts`.

Reply:
fixed